Header header
uint32 cycle
uint32 gas_id
float32 concentration
float32 temp
float32 pres
float32 rel_hum
float32 abs_hum